db/products.js

```diff
@@ -0,0 +1,88 @@
+(function () {
```

Comment:
In backend files, you do not need to create a closure through an IIFE like this anymore. Each file is in their own closure. And the only things exposed to other files is on module.exports.